observability/network_observability/netobserv_cli/netobserv-cli-overview.adoc

```diff
@@ -0,0 +1,9 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="netobserv-cli-overview"]
+= Network Observability CLI overview
+include::_attributes/common-attributes.adoc[]
+:context: cli-overview
+
+toc::[]
+
+include::modules/network-observability-netobserv-cli-about.adoc[leveloffset=+1]
```

Comment:
Thanks for questioning this Andrea. I actually originally had the Overview info in the beginning of the Installation assembly, but then I kept waffling like, this info is really overviewy and not focused exactly on the Installation user story. But I agree, the Overview is really underwhelming and I noticed other CLI plugins put their Overviews with the Installation. I think I'll move it back because your question is validating my initial instinct to have it in the Installation assembly. And specifically, I'm struggling with having enough intro text in the Overview assembly to accompany the overview module.